ReadMe.md

```diff
@@ -1,4 +1,4 @@
-## Artificial&nbsp;Intellegence Computer&nbsp;Science&nbsp;Club
+## Bakersfield&nbsp;College&nbsp;Computer&nbsp;Science&nbsp;Club
```

Comment:
It might be a good idea to allow a linebreak _somewhere_ in there

![Preview](https://cloud.githubusercontent.com/assets/251545/19199570/491bcd92-8c8a-11e6-82b5-f28ab5833cc1.PNG)